includes/mailchimp.admin.inc

```diff
@@ -68,3 +68,57 @@ function mailchimp_clear_list_cache_form_submit($form, &$form_state) {
   mailchimp_get_lists(array(), TRUE);
   drupal_set_message(t('MailChimp List cache cleared.'));
 }
+
+/**
+ * Menu callback to access mailchimp account info
+ * This allows developers to easily figure out
+ * the account_id associated with a given key,
+ * enabling support for multiple api keys.
+ * @see hook_mailchimp_api_key().
+ */
+function mailchimp_account_overview_page() {
+  $accounts = array();
+  $output = '<h2>Mailchimp API Account Info</h2>';
+  $api_key_form = drupal_get_form('mailchimp_account_retrieval_form');
+  $output .= drupal_render($api_key_form);
+  if (isset($_REQUEST['mailchimp_api_key'])) {
```

Comment:
Thanks @Greg-Boggs. Since these pages are administrative in nature and using access arguments I hadn't thought of it. Better safe than sorry, however, so I've updated the PR accordingly.